### ISSUE TITLE
Count Calls before Rating

### ISSUE THREAD

**Issue by danisam** [cgrates/cgrates#28]
Hello,
we need availability to Count calls per destination for a clients before a Rating plan is apply to this client.
for example the client1 has 1000 Locals Calls free/month  before we start billing them the calls using his profile.

Regards

Daniel

**Comment by danbogos** [comment]
This could go hand in hand with cost based on CDR stats, eg: if ASR > 70%, apply a rate, otherwise another. Something to postpone for a future implementation. There is also the right place to keep track of number of calls or other counters.

DanB

**Comment by rif** [comment]
You can solve this by adding a high weight balance with 1000 units having a special rate of 0 with connection fee of 1 unit.
In this way each call would only take 1 unit from this balance and after 1000 calls this balance will be depleted and the engine will start using other balances for billing.

Please tell us if this is a solution for your issue.

**Comment by danbogos** [comment]
Since there was no activity/reply anymore, we consider this ticket fixed.

DanB
